pkg/volume/emptydir/empty_dir_test.go

```diff
@@ -21,20 +21,18 @@ package emptydir
 
 import (
```

Comment:
Also this needs to be removed: https://github.com/kubernetes/kubernetes/blob/c15581b277e9e42d830898357fdbd92dcf7981c6/pkg/volume/emptydir/empty_dir.go#L115

Reply:
I thought in two releases I can remove the feature gate.

Reply:
It is locked to true. So you don't need this check. Ideally, removal of feature gate do not touch any code besides the definition of a feature gate itself

Reply:
Removed.